compiler/src/dmd/link.d

```diff
@@ -1101,7 +1101,7 @@ public DArray!ubyte runPreprocessor(ref const Loc loc, const(char)[] cpp, const(
                  */
                 OutBuffer buf;
                 buf.writestring(cpp);
-                buf.printf(" /P /Zc:preprocessor /PD /nologo %.*s /FI%s /Fi%.*s",
+                buf.printf(" /P /Zc:preprocessor /PD /nologo /utf-8 %.*s /FI%s /Fi%.*s",
```

Comment:
If you don't mind, this would make a great separate PR. We can merge it right away, and if there's an issue with it it will not affect the rest of this PR.

Reply:
If there is an issue with it, we'll have a massive problem on our hands.

Our lexer only supports UTF-8 and without it, Unicode identifiers are not working with ImportC for MSVC as I found out.

But yeah I guess I can split it out.

Reply:
thank you